mksh build script checks that sizeof(size_t) == sizeof(void *). this is not the
case on 9front/amd64, where sizeof(size_t) == 4 and sizeof(void *) == 8.

--- a/Build.sh  Fri May 15 03:51:24 2015
+++ b/Build.sh	Fri May 15 03:51:24 2015
@@ -2162,8 +2162,6 @@
 cta(ari_size_no_matter_of_signedness, sizeof(mksh_ari_t) == sizeof(mksh_uari_t));
 
 cta(sizet_size_no_matter_of_signedness, sizeof(ssize_t) == sizeof(size_t));
-cta(sizet_voidptr_same_size, sizeof(size_t) == sizeof(void *));
-cta(sizet_funcptr_same_size, sizeof(size_t) == sizeof(void (*)(void)));
 /* our formatting routines assume this */
 cta(ptr_fits_in_long, sizeof(size_t) <= sizeof(long));
 cta(ari_fits_in_long, sizeof(mksh_ari_t) <= sizeof(long));
